### PR TITLE
Bratseth/minor improvements

### PR DESCRIPTION
**Pull request by bratseth**
@lesters please review

I wonder how we ended up with a basic-search-java which has nothing whatsoever to do with basic-search or any other kind of search ...

### CODE REVIEW

File: basic-search-java/src/main/application/searchdefinitions/music.sd [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# Copyright 2017 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
```
Comment (lesters) [inline basic-search-java/src/main/application/searchdefinitions/music.sd]: 2018

File: basic-search-java/src/main/java/com/mydomain/example/ExampleSearcher.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
+// Copyright 2017 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
```
Comment (lesters) [inline basic-search-java/src/main/java/com/mydomain/example/ExampleSearcher.java]: 2018

File: basic-search-java/src/test/java/com/mydomain/example/ExampleSearcherTest.java [hunk 3]
```diff
@@ -0,0 +1,50 @@
+// Copyright 2017 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
```
Comment (lesters) [inline basic-search-java/src/test/java/com/mydomain/example/ExampleSearcherTest.java]: 2018

File: generic-request-processing/README.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
+<!-- Copyright 2017 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root. -->
```
Comment (lesters) [inline generic-request-processing/README.md]: 2018

File: generic-request-processing/pom.xml [hunk 5]
```diff
@@ -0,0 +1,114 @@
+<?xml version="1.0"?>
+<!-- Copyright 2017 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root. -->
```
Comment (lesters) [inline generic-request-processing/pom.xml]: 2018

File: generic-request-processing/src/main/application/hosts.xml [hunk 6]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2017 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root. -->
```
Comment (lesters) [inline generic-request-processing/src/main/application/hosts.xml]: 2018

File: generic-request-processing/src/main/application/services.xml [hunk 7]
```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2017 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root. -->
```
Comment (lesters) [inline generic-request-processing/src/main/application/services.xml]: 2018

File: generic-request-processing/src/test/java/com/mydomain/example/ApplicationMain.java [hunk 8]
```diff
@@ -0,0 +1,33 @@
+// Copyright 2017 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
```
Comment (lesters) [inline generic-request-processing/src/test/java/com/mydomain/example/ApplicationMain.java]: 2018

File: generic-request-processing/src/test/java/com/mydomain/example/ApplicationTest.java [hunk 9]
```diff
@@ -0,0 +1,32 @@
+// Copyright 2017 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
```
Comment (lesters) [inline generic-request-processing/src/test/java/com/mydomain/example/ApplicationTest.java]: 2018

**Comment by bratseth** [comment]
Updated some years ...

**Comment by bratseth** [comment]
@jobergum fyi

**Comment by jobergum** [comment]
Great!

**Pull request merged**